src/Keboola/DbExtractor/Configuration/ConfigDefinition.php

```diff
@@ -113,6 +113,7 @@ public function addSshNode()
                 ->scalarNode('remotePort')->end()
                 ->scalarNode('localPort')->end()
                 ->scalarNode('user')->end()
+                ->booleanNode('compression')->defaultValue(false)->end()
```

Comment:
MySQL extractor pouziva kompresi na urovni DB spojeni. Tam se ten parametr uz jmenuje `networkCompression`. Mohlo by to byt asi stejne?

https://github.com/keboola/db-extractor-mysql/blob/master/src/Keboola/DbExtractor/Configuration/MySQLConfigDefinition.php#L37

Reply:
ale ten mysql networkCompression je option DB connection a tohle je option ssh connection tak nejsou uplnej stejne ne?

Reply:
bude to mozny pro mysql user po tom pouzivat oba, ten db option i tohle do hromady?

Reply:
Ano tam se to pouziva v DB, ne v SSH configuraci. Ale je to taky DB extractor tak by ty parametry mohly byt pojmenovane stejne.

Reply:
bohuzel nemuzem, v ssh-tunnel lib to se jmenuje `compression`.  muzem pouzivat stejne labelu v UIcko

Reply:
treba?

```php
$sshConfig['compression'] = $sshConfig['networkCompression']
```